components/usage-api/usage/v1/billing.proto

```diff
@@ -22,6 +24,11 @@ service BillingService {
 
   rpc CreateStripeCustomer(CreateStripeCustomerRequest) returns (CreateStripeCustomerResponse) {};
 
+  rpc CreateStripeSubscription(CreateStripeSubscriptionRequest) returns (CreateStripeSubscriptionResponse) {};
+
+  rpc SetDefaultPaymentMethodForCustomer(SetDefaultPaymentMethodForCustomerRequest) returns (SetDefaultPaymentMethodForCustomerResponse) {};
+
+  rpc FindUncancelledSubscriptionByAttributionId(FindUncancelledSubscriptionByAttributionIdRequest) returns (FindUncancelledSubscriptionByAttributionIdResponse) {};
```

Comment:
What's the use-case for this RPC? I believe it's just an implementation detail of CreateStripeSubscription.

Reply:
This finds a subscription ID and is called in two places: 1) entitlement service to determine parallel workspaces, etc., and 2) when deleting a team.

Reply:
Okay, in this case, let's call it `GetStripeSubscription`. If it's cancelled, it can be a property of the resulting StripeSubscription. This avoid pushing properties of the result into the RPC name (better to call ListFurniture, than ListRedChairsWithPolkaDotCovers), makes it more general and allows for more flexibility.

Reply:
Alternatively, you can also add `"include_cancelled = bool"` as a property of the request object to specify what you want to filter on.

Reply:
> Alternatively, you can also add `"include_cancelled = bool"` as a property of the request object to specify what you want to filter on.

I'm actually unsure why this was called "find uncancelled" because the API call doesn't pass any extra params or filter cancelled results. Let's keep it as is for now but renamed.

Reply:
Or if it can return a list of results, then doing `ListSubscriptions`. As for the arguments passed, see `GetStriepCustomer` on how it allows to pass in Attribution ID, or StripeCustomerID